test/runTest/util.js

```diff
@@ -102,12 +102,12 @@ module.exports.fetchVersions = function (isNighlty) {
     return new Promise((resolve, reject) => {
         https.get(
             isNighlty
```

Comment:
It seems we do not use `nightly`.

Reply:
I will change nothing here

Reply:
https://github.com/appflow-ai/echarts/blob/master/test/runTest/server.js#L234

I saw you delete the yml config of punish nightly version but It depend on nightly version somewhere?

Reply:
actually I just don't to run night job, it do not mean it need to be removed